services/ad-tech/src/controllers/contextual-auction-runner.ts

```diff
@@ -40,66 +40,40 @@ export interface ContextualBidResponse {
  */
 export const ContextualAuctionRunner = (() => {
   /** Helper function to construct contextual bid URLs. */
-  const getBidUrlByBidder = (
+  const getContextualBidUrls = (
     bidderHosts: string[],
     signals: {[key: string]: string},
-  ): Map<string, string> => {
+  ): string[] => {
     // Convert signals into request query parameters.
     const bidQuery = Object.entries(signals)
       .map(([key, value]) => `${key}=${value}`)
       .join('&');
-    // Assemble DSP contextual bidding endpoints.
-    const bidUrlByBidder = new Map<string, string>();
-    for (const bidderHost of bidderHosts) {
-      bidUrlByBidder.set(
-        /* bidder= */ new URL(
-          `https://${bidderHost}:${EXTERNAL_PORT}/`,
-        ).toString(),
-        /* contextualBidUrl= */ new URL(
-          `http://${bidderHost}:${PORT}/dsp/contextual-bid?${bidQuery}`,
-        ).toString(),
-      );
-    }
-    return bidUrlByBidder;
-  };
-
-  /** Returns contextual bid response for the given URL. */
-  const getContextualBidResponse = async (
-    bidder: string,
-    bidUrl: string,
-  ): Promise<ContextualBidResponse> => {
-    console.log('Making contextual bid request', {bidUrl});
-    const response = await fetch(bidUrl);
-    if (!response.ok) {
-      console.log('Error in contextual bid response', {
-        bidUrl,
-        statusText: response.statusText,
-      });
-      return {
-        bidder,
-        bid: '0.0',
-      };
-    }
-    const bidResponse = await response.json();
-    console.log('Received bid response from DSP', {bidResponse});
-    if (!bidResponse.bidder) {
-      // Set bidder if missing.
-      bidResponse.bidder = bidder;
-    }
-    return bidResponse;
+    return bidderHosts.map((bidderHost) =>
+      new URL(
+        `http://${bidderHost}:${PORT}/dsp/contextual-bid?${bidQuery}`,
```

Comment:
replace with DSP_URI, DSP_A_URI and DSP_B_URI (because docker is using http port 8080, while cloud run uses https port 443)

Reply:
@siddharth-sahoo where did this logic go? Trying to follow in [this commit](https://github.com/privacysandbox/privacy-sandbox-demos/pull/425/commits/025ef05c5911a8b084efe85362d733df46e08514). Was `contextual-auction-runner.ts` broken up into `contextual-auction-helper.ts` and the buyer/seller contextual routers?